cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -2087,6 +2095,61 @@ declare namespace Deno {
       options?: KvListOptions,
     ): KvListIterator<T>;
 
+    /**
+     * Add a value into the database queue to be delivered to the queue
+     * listener via {@linkcode Deno.Kv.queueListen}.
+     *
+     * ```ts
+     * const db = await Deno.openKv();
+     * await db.enqueue("bar");
+     * ```
+     *
+     * The `delay` option can be used to specify the delay (in milliseconds)
+     * of the value delivery. The default delay is 0, which means immediate
+     * delivery.
+     *
+     * ```ts
+     * const db = await Deno.openKv();
+     * await db.enqueue("bar", { delay: 60000 });
+     * ```
+     *
+     * The `keys_if_undelivered` option can be used to specify the keys to
+     * be set if the value is not successfully delivered to the queue
+     * listener after several attempts. The values are set to the value of
+     * the queued message.
+     *
+     * ```ts
+     * await db.enqueue("bar", { keys_if_undelivered: ["foo", "bar"] });
+     * ```
+     */
+    enqueue(
+      value: unknown,
+      options?: { delay?: number; keys_if_undelivered?: Deno.KvKey[] },
```

Comment:
`keys_if_undelivered` should be camel case.

Reply:
thanks, fixed.